examples/jazz-sveltekit/src/routes/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import { beforeNavigate, goto } from "$app/navigation";
+  import { Account } from "jazz-tools";
+  import { AccountCoState } from "jazz-tools/svelte";
+
+  const account = new AccountCoState(Account, {
+    resolve: {
+      profile: true,
+    },
+  });
+  const me = $derived(account.current);
+  const navigate = async (route: string) => {
+    // Flush the name to avoid SSR picking up the previous name
+    await me?.profile.$jazz.waitForSync();
```

Comment:
Ugh, this is annoying. Can you explain the error you saw? I feel like this is something that we should address on our end

Reply:
GCO-867 to potentially pick up in cooldown as discussed